
Compile fixes x86-64 - imported from ROCK Linux.

  - Rene Rebe <rene@exactcode.de>

diff -urN bin86-0.16.14-orig/ld/x86_aout.h bin86-0.16.14/ld/x86_aout.h
--- bin86-0.16.14-orig/ld/x86_aout.h	2004-11-07 20:07:31.506442000 +0100
+++ bin86-0.16.14/ld/x86_aout.h	2004-11-07 20:07:41.106982496 +0100
@@ -12,6 +12,9 @@
 #if defined(i386) || defined(__BCC__) || defined(MSDOS)
 typedef long Long;
 #define __OUT_OK 1
+#elif defined(__x86_64__)
+typedef int Long;
+#define __OUT_OK 1
 #else
 typedef char Long[4];
 #endif
